lib/coqProject_file.ml

```diff
@@ -251,39 +251,39 @@ let process_cmd_line ~warning_fn orig_dir proj args =
     aux { proj with defs = proj.defs @ [sourced (v,def)] } r
   | "-arg" :: a :: r ->
     aux { proj with extra_args = proj.extra_args @ List.map sourced (process_extra_args a) } r
-  | f' :: r ->
-      let f = CUnix.correct_path f' orig_dir in
+  | f :: r ->
+      let abs_f = CUnix.correct_path f orig_dir in
```

Comment:
@Columbus240 Notice that I renamed the original `f'` to `f`, so the `check_filename f` behaves differently.  Right?

> If I'm not badly mistaken

Questions from others, even if mistaken, can be quite helpful.  So no need to apologize with "badly".  I usually make several mistakes before breakfast each day.

Reply:
Both before and after the change, the code does the following (if I interpret it correctly): Take the first element of the list, and apply `CUnix.correct_path _ orig_dir` to it. Before the change, the first element of the list was called `f'` and the result was stored in `f`, after the change the first element of the list is called `f` and the result is stored in `abs_f`.
So I don’t see the functional change, because the renaming was done consistently.

Reply:
Of course `abs_f` is a slightly more evocative name, making the meaning of this variable a little more clear.

Reply:
>So I don’t see the functional change, because the renaming was done consistently.

not quite, see eg line 259 (before PR) https://github.com/coq/coq/pull/15775/files#diff-13482a4486eee3b60cca52baee0cf046a6d89d7d009e314364bce73ee4377effL259

Reply:
Ooh, that makes sense. Thanks for the explanation.